codecov.yml

```diff
@@ -15,5 +15,7 @@ coverage:
     # changes tells us if there are unexpected code coverage changes in other files
     # which were not changed by the diff
     changes: false
```

Comment:
Yes, because you  might write tests that touch a package that you didn't change, and the coverage for that package would change, but there were no changes in that package itself.

I think if this were enabled, codecov would add a separate check that would fail.  The three checks that codecov can do are project (we want to gate on total project coverage), patch (that allows gating based on what % of the diff is covered by tests), and failing if there are unexpected changes. 

We're only enabling the first check.